io/eolearn/tests/test_raster_io.py

```diff
@@ -383,8 +383,15 @@ def test_export_import_sequence(no_data_value, data_type):
         export_task.execute(eopatch=eopatch, filename=filename)
 
         with rasterio.open(file_path) as src:
-            tif_array = src.read(masked=True)
-            assert np.sum(tif_array.mask) == np.sum(np_arr == no_data_value)
+            # when reading, move axis to have the tif_array in the EOPatch.data_timeless dimension structure
+            tif_array = np.moveaxis(src.read(masked=True), 0, -1)
+
+            if no_data_value is not np.nan:
+                no_data_arr = np_arr == no_data_value
+            else:
+                no_data_arr = np.isnan(np_arr)
```

Comment:
would `no_data_arr = np.isnan(np_arr) if no_data_value is np.nan else np_arr == no_data_value` also work?

Reply:
Then I would just change the order of the branches, so that the condition would be `no_data_value is np.nan`.